src/primitives/Lazy.tsx

```diff
@@ -64,6 +70,32 @@ function createLazy<T>(
     queueMicrotask(() => viewRef.scrollToIndex(index));
   }
 
+  function isInNonScrollableZone(
+    this: lngp.NavigableElement,
```

Comment:
Does this need to be slightly different? Can we make this one function and put it in withScrolling?